src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -244,10 +244,10 @@ int InvokeDelegate(host_main_fn host_main, int argc, const char** argv, const bo
 
 std::string get_os_version()
 {
-    std::vector<std::string> qualifiers { "DISTRIB_ID=", "DISTRIB_RELEASE=" };
+    std::vector<std::string> qualifiers { "ID=", "VERSION_ID=" };
```

Comment:
because we always expect `"` you could change the qualifiers to contain the leading `"` and then just copy length-1 chars.